src/plugins/mf_priority.cpp

```diff
@@ -114,7 +116,7 @@ int64_t priority_calculation (flux_plugin_t *p,
     priority = round ((fshare_weight * fshare_factor) +
                       (queue_weight * queue_factor) +
                       (bank_weight * bank_factor) +
-                      (urgency - 16));
+                      (urgency_weight * (urgency - 16)));
```

Comment:
Not strictly related to this PR so feel free to ignore but FLUX_JOB_URGENCY_DEFAULT could be used in place of the value 16 here.

Reply:
This is a great suggestion - definitely makes this more clear. I'll go ahead and push up an extra commit to use this macro instead of just 16.